step_04/ReadMe.md

```diff
@@ -53,16 +56,16 @@ Then to introduce Cycles, you need to set the frequency (predefined attribute) o
 
         ...
 
-        scenario_cfg = Config.configure_scenario(id="my_scenario",
-                                                 pipeline_configs=[pipeline_cfg],
-                                                 frequency=Frequency.MONTHLY)
+        scenario_cfg = Config.configure_scenario_from_tasks("my_scenario",
+                                                            [pipeline_cfg],
+                                                            frequency=Frequency.MONTHLY)
 
 
         ```
 
 
 
-As you can see, a Cycle is activated once you have set the desired frequency on the scenario. In this code snippet, since we have specified `frequency=Frequency.MONTHLY`, the corresponding scenario will be automatically attached to the correct period (month) once it is created. The _creation_date_ here is artificially given to the scenarios.
+As you can see, a Cycle is activated once you have set the desired frequency on the scenario. In this code snippet, since we have specified `frequency=Frequency.MONTHLY`, the corresponding scenario is automatically attached to the correct period (month) once it is created. _creation_date_ is automatically assigned `datetime.datetime.now()` but, we are here manually overriding it. A Scenario belongs to a Cycle based on the _creation_date_ of the scenario. It can be "attached" to a specific cycle by manually setting its _creation_date_.
```

Comment:
```suggestion
As you can see, a Cycle is activated once you have set the desired frequency of the scenario. In this code snippet, since we have specified `frequency=Frequency.MONTHLY`, the corresponding scenario is automatically attached to the correct period (month) when it is created. 

The Cycle which a Scenario belongs to is based on the _creation_date_ (`datetime.datetime.now()` by default) of the scenario. It can be "attached" to a specific cycle by manually setting its _creation_date_, as we are doing in the following example.
```